front/tests/pages/common-page-model.ts

```diff
@@ -0,0 +1,22 @@
+import { Page } from '@playwright/test';
+import rollingstockTranslation from '../../public/locales/fr/rollingstock.json';
+
+class PlaywrightCommonPage {
+  readonly rollingstockTranslation: typeof rollingstockTranslation;
+
+  constructor(readonly page: Page) {
+    this.rollingstockTranslation = rollingstockTranslation;
+    this.page = page;
+  }
+
+  getRollingstockTranslation(key: keyof typeof rollingstockTranslation) {
+    return this.rollingstockTranslation[key];
+  }
+
+  async setTag(tag: string) {
```

Comment:
Update this function to set multiple tags
```suggestion
  async setTags(tags: string[]) {
```